mods/ra/maps/soviet-05/soviet05.lua

```diff
@@ -171,7 +171,7 @@ Tick = function()
 
 	if not RCheck then
 		RCheck = true
-		if Difficulty == "easy" and ReinfCheck then
+		if Difficulty == "easy" and RCheck then
```

Comment:
The additional checking of `RCheck` here makes no sense as the line above sets it to `true`. If we're sure that is the intended variable please just remove it from the `if`.